src/components/integrated/OakLessonLayout/OakLessonLayout.tsx

```diff
@@ -123,9 +119,9 @@ function pickSectionColours(
   switch (sectionName) {
     case "overview":
       return [
-        "bg-decorative4-subdued",
+        "bg-decorative1-main",
         "bg-primary",
-        "border-decorative4",
+        "border-decorative1-stronger",
         "bg-primary",
```

Comment:
The overview page is no longer violet, but green — this is to soften the transition from the old design.